src/plugins/intel_gpu/src/kernel_selector/cl_kernels/broadcast_gpu_ref.cl

```diff
@@ -379,7 +408,14 @@ KERNEL(broadcast_gpu_ref)(
         const uint out_pos = OUTPUT_GET_INDEX(out_b, out_f, out_y, out_x);
         const uint idx_pos = FUNC_CALL(get_idx_pos)(OPTIONAL_SHAPE_INFO_TENSOR out_b, out_f, out_y, out_x);
 #endif
-        output[out_pos] = TO_OUTPUT_TYPE(input[idx_pos]);
+        #if HAS_FUSED_OPS
+            uint offset = 0;
```

Comment:
nit: what about `const uint offset =0`?

Reply:
I will update this in next uploading PR.